jmclient/jmclient/taker.py

```diff
@@ -351,8 +353,9 @@ def prepare_my_bitcoin_data(self):
             jlog.debug("Estimated ins: "+str(est_ins))
             est_outs = 2*self.n_counterparties + 1
             jlog.debug("Estimated outs: "+str(est_outs))
-            self.total_txfee = estimate_tx_fee(est_ins, est_outs,
-                                            txtype=self.wallet_service.get_txtype())
+            self.total_txfee = estimate_tx_fee(
+                est_ins, est_outs, txtype=self.wallet_service.get_txtype(),
+                outtype=self.wallet_service.get_outtype(self.coinjoin_address()))
```

Comment:
See https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/921/files#r701344743

Reply:
Provided justification on the thread